aptos-move/framework/aptos-framework/sources/storage_gas.move

```diff
@@ -0,0 +1,254 @@
+module aptos_framework::storage_gas {
```

Comment:
kinda think "state_storage" explicitly is better.. as we did with the original "state_storage" module. But yeah Move doesn't have other "storage"s, the ambiguous part here is actually "state"